DS-Assignment1/Question 3&4/src/com/metacube/doublylinkedlist/Node.java

```diff
@@ -0,0 +1,58 @@
+package com.metacube.doublylinkedlist;
+@SuppressWarnings("rawtypes")
+public class Node<E extends Comparable<E>> {
```

Comment:
This node can be extended version of singly linked list node, just previousNode is extra here.